apps/web-giddh/src/app/settings/Trigger/setting.trigger.component.ts

```diff
@@ -144,50 +135,79 @@ export class SettingTriggerComponent implements OnInit, OnDestroy {
+                }
+            } else {
+                if(this.commonLocaleData?.app_something_went_wrong) {
+                    this.toaster.clearAllToaster();
+                    this.toaster.errorToast(this.commonLocaleData?.app_something_went_wrong);
+                }
+            }
+            this.isLoading = false;
         });
     }
 
     public onSubmit(data) {
         let dataToSave = cloneDeep(data);
         dataToSave.action = 'webhook';
         if (!dataToSave.name) {
-            this._toaster.errorToast(this.localeData?.validations?.trigger_name, 'Validation');
+            this.toaster.errorToast(this.localeData?.validations?.trigger_name, this.localeData?.validation);
             return;
         }
         if (!dataToSave.entity) {
-            this._toaster.errorToast(this.localeData?.validations?.entity_type, 'Validation');
+            this.toaster.errorToast(this.localeData?.validations?.entity_type, this.localeData?.validation);
             return;
         }
         if (!dataToSave.entityUniqueName) {
-            this._toaster.errorToast(this.localeData?.validations?.entity, 'Validation');
+            this.toaster.errorToast(this.localeData?.validations?.entity, this.localeData?.validation);
             return;
         }
         if (!dataToSave.scope) {
-            this._toaster.errorToast(this.localeData?.validations?.scope, 'Validation');
+            this.toaster.errorToast(this.localeData?.validations?.scope, this.localeData?.validation);
             return;
         }
         if (!dataToSave.filter) {
-            this._toaster.errorToast(this.localeData?.validations?.filter, 'Validation');
+            this.toaster.errorToast(this.localeData?.validations?.filter, this.localeData?.validation);
             return;
         }
         if (!dataToSave.action) {
-            this._toaster.errorToast(this.localeData?.validations?.action, 'Validation');
+            this.toaster.errorToast(this.localeData?.validations?.action, this.localeData?.validation);
             return;
         }
         if (!dataToSave.value && this.newTriggerObj.scope !== 'closing balance') {
-            this._toaster.errorToast(this.localeData?.validations?.enter_value, 'Validation');
+            this.toaster.errorToast(this.localeData?.validations?.enter_value, this.localeData?.validation);
             return;
         } else {
             delete dataToSave['value'];
         }
         if (!dataToSave.url) {
-            this._toaster.errorToast(this.localeData?.validations?.enter_url, 'Validation');
+            this.toaster.errorToast(this.localeData?.validations?.enter_url, this.localeData?.validation);
             return;
         }
-        this.store.dispatch(this._settingsTriggersActions.CreateTrigger(dataToSave));
+        
+        this.settingsTriggersService.CreateTrigger(dataToSave).pipe(takeUntil(this.destroyed$)).subscribe(response => {
+            this.toaster.clearAllToaster();
```

Comment:
Consider moving this block to a new method that will take the `response` and success message as argument for `create`, `update` and `delete` trigger scenarios